web/source/build-web-core.sh

```diff
@@ -0,0 +1,19 @@
+#! /bin/bash
+# 
+# Compile KeymanWeb's 'keyboard-processor' module, one of the components of Web's 'core' module.
+#
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+# This script runs from its own folder
+cd "$(dirname "$THIS_SCRIPT")"
+
+# Generates a linkable TS file; defined in resources/build-utils.sh.
+exportEnvironmentDefinitionTS
+
+# Compile web's `keyboard-processor` module.
+npm run tsc -- -p source/web-core.tsconfig.json
```

Comment:
Where in the build process does `build-web-core.sh` get called?

When the dust settles, can you update web/README.md if anything changes?

Reply:
It doesn't.  Right now, it's just a manual call.